BP/history/provinces/bp_k_egypt.txt

```diff
@@ -1,8 +1,16 @@
-﻿# k_egypt
+﻿# Notes: 
+# Cyrenica (really the Pentapolis) was a Greek and I can't find any reference to it becoming coptic before the Arab migrations
+# Kemetism dies out fairly soon; Philae is the last holdout. The temple was closed in 553 by Justinian I
+# Siwa and Philae (Aswan) were the last basions of Kemetism
+# 600.1.1: Siwa converts to miaphysitism
```

Comment:
Ok on Philae and Siwa specifically being holdouts together with the two enclaves in the north, but in the south you have turned entire duchies Kemetic, while it was one province in WTWSMS. Shouldn't Miaphysite be more prevalent?

Reply:
No; the Kingdom of Kush (which was the old_nubian culture/Meroitic dynasty of Egypt) extended as far south as Meroë/c_abwab and Kemetism was adopted as the faith of that dynasty. As such, Kemetism proper is the more accurate faith for Nobatia and Makuria than Kushitism/Mandulis.

There was certainly some Miaphysite/Monophysite activity in the far north of Nubia, I don't think it would have extended that far south; for example, the Diocese of Faras wasn't established until the early VIIth century. While some rulers embraced Nicene/Miaphysite Christianity, it probably didn't reach the populous until later.

The rise of Miaphysitism in the region took place later, during the mid-late VIth and early VIIth centuries, which I've modeled in both the character and province histories.